<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CampusCard</class>
 <widget class="QWidget" name="CampusCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>357</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>357</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CampusCard</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校园卡模拟系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="CardControl">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>卡片管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CanteenApp">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>食堂应用</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Summary">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>汇总分析</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="inputAllBtn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>导入批量数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>7</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>9</month>
     <day>4</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd hh:mm:ss:zzz</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
